package.json

```
@@ -12,7 +12,7 @@
     "format:package_json": "lerna exec -- prettier-package-json",
     "postinstall": "husky",
     "lint": "npm run lint:js && npm run lint:md",
-    "lint:js": "eslint .storybook/ packages/ utils/ --ext js,mjs,ts,tsx --max-warnings 0",
+    "lint:js": "eslint . --max-warnings 0",
```

```suggestion
    "lint:js": "eslint --max-warnings 0",
```

...is the same with eslint v9.